decompose-router/src/commonMain/kotlin/io/github/xxfast/decompose/router/Router.kt

```diff
@@ -128,3 +129,42 @@ fun <T : Instance, C: @Serializable Any> rememberOnRoute(
 
   return instance
 }
+
+/***
+ * Retrieve a instance of [V] that is scoped to the current route
+ *
+ * @param type of [V] instance
+ * @param childClass class of stack child[T]
+ * @param keyBlock lambda to configure the key [which was used to save the instance [T]] by default [type].key
+ */
+
+inline fun <C : @Serializable Any, reified T : Any, reified V : Instance> Router<C>.getInstanceFromBackStack(
+  type: KClass<V>,
+  childClass: KClass<T>,
+  block: ((childConfiguration: T) -> Any?) = { null }
+): V {
+  val child: Child.Created<C, RouterContext>? = this.stack.value.backStack
```

Comment:
`backStack` doesn't include the active child. Perhaps `items` is what is actually meant here?